ui/components/Flex.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import styled from "styled-components";
+
+type Props = {
+  className?: string;
+  column?: boolean;
+  align?: boolean;
+  between?: boolean;
+  center?: boolean;
+  wide?: boolean;
+  wrap?: boolean;
+};
+
+const Styled = (component) => styled(component)`
+  display: flex;
+  flex-direction: ${(props) => (props.column ? "column" : "row")};
+  align-items: ${(props) => (props.align ? "center" : "start")};
+  ${({ between }) => between && "justify-content: space-between"};
+  ${({ center }) => center && "justify-content: center"};
+  ${({ wide }) => wide && "width: 100%"};
+  ${({ wrap }) => wrap && "flex-wrap: wrap"};
+  ${({ end }) => end && "justify-content: flex-end"};
+`;
+
+class Flex extends React.PureComponent<Props> {
```

Comment:
This is great, I forgot to give it a go after you sent it to me *face palm*! Could we have it added to the `ui-components` library do you think?